src/classloader/opcode_parser.rs

```diff
@@ -409,27 +622,52 @@ pub fn parse_opcodes<'a>(
             175 => {
                 opcodes.push(OpCode::Dreturn);
             },
+            176 => {
+                opcodes.push(OpCode::Areturn);
+            },
             177 => {
                 opcodes.push(OpCode::Return);
             },
             178 => {
-                let (new_content, opcode) = parse_getstatic(
+                let (new_content, field) = parse_static_field(
                     current_content,
                     class_file,
                     runtime_cp,
                     heap,
                 )?;
-                opcodes.push(opcode);
+                opcodes.push(OpCode::GetStatic(field));
                 current_content = new_content;
             },
+            179 => {
+                let (new_content, field) = parse_static_field(
+                    current_content,
+                    class_file,
+                    runtime_cp,
+                    heap,
+                )?;
+                opcodes.push(OpCode::PutStatic(field));
+                current_content = new_content;
+            },
+            180 => {
+                todo!(
+                    "GetField(Rc<dyn Any>), needs information
+                    on how to resolve fields at execution time"
```

Comment:
Only just now saw the CI formatting failure regarding this line.
You probably want the this block to look like this (todo! obviously staying as far indented as it already is, the focus is on the string wrapping):
```rust
    todo!(
        "GetField(Rc<dyn Any>), needs information \
    on how to resolve fields at execution time"
    )
```

This has the effect that:
- cargo fmt stops complaining
- the string is output as a single line, without a single whitespace where in the source code the line break is